examples/private_ai_studio/main.tf

```diff
@@ -55,16 +137,32 @@ module "aihub" {
   }
 
   container_registry = {
-    create_new     = true
-    zone_redundant = false
+    create_new = false
```

Comment:
Should this be set to `true` so a new instance of container registry is created?